<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExchangeMenu</class>
 <widget class="QWidget" name="ExchangeMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>456</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>692</width>
    <height>456</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>692</width>
    <height>456</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="exchange_bgrd_lbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>456</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding-right: 15px; /* make room for the arrows */
border-image: url(:/Pyramid/fon_next_menu.png);
border-width: 3;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="send_track_but">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>173</width>
     <height>152</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton:hover:pressed {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/send_track_n.png);
     border-width: 3;}
 
QPushButton{
     padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/send_track.png);
     border-width: 3;}

 QPushButton:hover {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/send_track.png);
     border-width: 3;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="send_photo_but">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>173</width>
     <height>152</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton:hover:pressed {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/send_photo_n.png);
     border-width: 3;}
 
QPushButton{
     padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/send_photo.png);
     border-width: 3;}

 QPushButton:hover {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/send_photo.png);
     border-width: 3;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="get_track_but">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>173</width>
     <height>152</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton:hover:pressed {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/get_track_n.png);
     border-width: 3;}
 
QPushButton{
     padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/get_track.png);
     border-width: 3;}

 QPushButton:hover {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/get_track.png);
     border-width: 3;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="get_paletka_but">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>173</width>
     <height>152</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton:hover:pressed {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/get_paletka_n.png);
     border-width: 3;}
 
QPushButton{
     padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/get_paletka.png);
     border-width: 3;}

 QPushButton:hover {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/get_paletka.png);
     border-width: 3;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textStatusLbl">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>статус:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="statusLbl">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewVideoBut">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>173</width>
     <height>152</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton:hover:pressed {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/video_otpravit_n.png);
     border-width: 3;}
 
QPushButton{
     padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/video_otpravit.png);
     border-width: 3;}

 QPushButton:hover {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/video_otpravit.png);
     border-width: 3;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="sendMarks">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>260</y>
     <width>173</width>
     <height>152</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton:hover:pressed {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/otprav_otm_n.png);
     border-width: 3;}
 
QPushButton{
     padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/otprav_otm.png);
     border-width: 3;}

 QPushButton:hover {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/otprav_otm.png);
     border-width: 3;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Resources/PyramidWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
